tests/Unit/ControlSystem/Test_UpdateFunctionOfTime.cpp

```diff
@@ -163,4 +189,347 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.UpdateFunctionOfTime",
     CHECK(cache_quatfot == expected_quatfot);
   }
 }
+
+void test_update_aggregator() {
+  // Notice the combined name has an extra state that isn't active
+  const std::string combined_name{
+      "AlaskaCaliforniaFloridaIllinoisTexasWisconsin"};
+  std::unordered_set<std::string> names{"Illinois", "California", "Wisconsin",
+                                        "Florida", "Texas"};
+  std::unordered_map<std::string, DataVector> signals{};
+  signals["Wisconsin"] = DataVector{5.0};
+  signals["Texas"] = DataVector{4.0};
+  signals["Illinois"] = DataVector{3.0};
+  signals["Florida"] = DataVector{2.0};
+  signals["California"] = DataVector{1.0};
+  auto expected_signals = signals;
+
+  control_system::UpdateAggregator unserialized_aggregator{combined_name,
+                                                           names};
+  control_system::UpdateAggregator aggregator =
+      serialize_and_deserialize(unserialized_aggregator);
+
+  CHECK(aggregator.combined_name() == combined_name);
+
+  aggregator.insert("Wisconsin", DataVector{1.0}, 10.0,
+                    std::move(signals.at("Wisconsin")), 5.0);
```

Comment:
[optional] Once you change the rvalue reference argument these can be non-moved and then you don't need two copies of the signals map.  (This is why rvalue reference arguments are annoying.)